packages/create-playkit-plugin/createPlaykitPlugin.js

```diff
@@ -152,43 +205,57 @@ function printValidationResults(results) {
     }
 }
 
-const hiddenProgram = new commander.Command()
-    .option(
-        '--internal-testing-template <path-to-template>',
-        '(internal usage only, DO NOT RELY ON THIS) ' +
-        'use a non-standard application template'
-    )
-    .parse(process.argv);
+(async () => {
+    if (typeof projectName === 'undefined') {
```

Comment:
the questions should be asked regardless if someone provided `projectName`. 

I will modify the jira ticket to better implement this part.
prior for running the prompts, check if user provided `projectName`. if it does, check if this is a path or only a name and use those values as initials for `projectName` and `projectDestination`.

- given `~/dev/temp/abcd` - `projectName=abcd, projectDestination=~/dev/temp/abcd`
- given `abcd` - `projectName=acbd, projectDestination=${process.cwd()}/playkit-js-abcd`